code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -11,7 +11,7 @@ info:
     It enables a Service Provider (SP) to verify the phone number of the mobile device being used to access their service where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope) either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
 
     # Resources and Operations overview
-    This API currently provides two endpoints where both require a **3-legged token** and authentication via **mobile network** (excluding for example by SMS/OTP or user/password as an authentication method):
+    This API currently provides two endpoints where both require either a **3-legged token** or **API Key** and authentication via **mobile network** (excluding for example by SMS/OTP or user/password as an authentication method):
```

Comment:
@a-ramit can we have a call to go through the flow? Can you open an issue inside Commonalities WG to discuss API-key topic.